examples/sycl/pvc/pvc_bfloat_dpas_gemm_cute.cpp

```diff
@@ -233,7 +233,7 @@ struct ExampleRunner {
     block_D.reset(M * N * L);
     block_ref_D.reset(M * N * L);
 
-    // TODO: Enable initialization on device directly once RNG is 
+    // TODO: Enable initialization on device directly once RNG is
```

Comment:
I think this is a noise

Reply:
Yes, it is just removing an empty space that should not be there anyway.